### PR TITLE
Adds link to MicroProfile Starter Extensions GitHub organization

### PR DESCRIPTION
**Pull request by Karm**

### CODE REVIEW

**Comment by Karm** [comment]
Let's not merge this as we have an ongoing debate about https://github.com/dev-tools-for-enterprise-java/

**Comment by Emily-Jiang** [comment]
The new org and vs code extension are https://github.com/dev-tools-for-enterprise-java/mp-starter-vscode-ext. The org name is undergoing approval and subject to change. Please hold on and I will confirm in due course.

**Comment by Emily-Jiang** [comment]
Please change the PR to add a link to https://github.com/MicroShed/mp-starter-vscode-ext

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-# microprofile-starter
+# MicroProfile Starter
 
 [![Gitter](https://badges.gitter.im/eclipse/microprofile-starter.svg)](https://gitter.im/eclipse/microprofile-starter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## IDE Extensions and tools
+
+Ecosystem of tools and IDE extensions is hosted at [MicroProfile Starter Extensions](https://github.com/microprofile-starter-extensions) GitHub organization.
```
Comment (Emily-Jiang) [inline README.md]: Plase link to https://github.com/MicroShed/mp-starter-vscode-ext and https://github.com/MicroShed/mp-starter-intellij-ext

Comment (rdebusscher) [reply README.md]: @Karm Can you redo the PR with latest URLs of the IDE extensions?

**Comment by Karm** [comment]
Out-of-date

**Pull request closed without merging**